src/index.js

```diff
@@ -168,38 +170,41 @@ class StickyTable extends React.Component {
       ['leftColumn', 'width', 'leftStickyColumnCount'],
       ['rightColumn', 'width', 'rightStickyColumnCount']
     ].forEach(([stickyKey, sizeKey, countPropKey]) => {
-      var insets = [0];
-      var count = props[countPropKey];
-      var netInset = 0;
+      var insets = [];
 
-      //HINT we only want this loop for the second sticky and up
-      for (s = 1; s < count; s++) {
-        var node = stickyKey === 'header' || stickyKey === 'leftColumn' ? cellNodes[0] : cellNodes[cellNodes.length - 1];
+      if (props[countPropKey] > 1) {
+        insets = [0]
+        var count = props[countPropKey];
+        var netInset = 0;
 
-        if (node) {
-          var boundingRect = node.getBoundingClientRect();
+        // HINT we only want this loop for the second sticky and up
+        for (s = 1; s < count; s++) {
+          var node = stickyKey === 'header' || stickyKey === 'leftColumn' ? cellNodes[s - 1] : cellNodes[cellNodes.length - s];
```

Comment:
this seems like the one significant change here - definitely should be doing something with s, and not sure how I missed that, but I'm pretty sure this isn't quite right. need to be doing something with rows and then cells both based on s (this is only accounting for more than 2 columns, but not more than 2 footers/headers)

Reply:
Hmm. You're right. This fixes columns but not the others.
Maybe getting the `rowNodes` from the table too? That way, if the `stickyKey` is `header` or `footer`, it should check against the cells (probably `rowNodes[s - 1] : rowNodes[rowNodes.length - s];?)